contracts/crosschain/ERC7786Aggregator.sol

```diff
@@ -216,9 +217,10 @@ contract ERC7786Aggregator is IERC7786GatewaySource, IERC7786Receiver, Ownable,
             // prevent re-entry
             tracker.executed = true;
 
+            // TODO: Should the messageId be empty here?
```

Comment:
Lets discuss that

Reply:
I propose to use `keccak256(abi.encode(nonce, msg.sender))`, where `nonce` is the one in the payload.

Reply:
Ah I see you're already using the `id`. That seems fine too.

Reply:
```suggestion
```